react-components/src/utilities/FdmSDK.ts

```diff
@@ -394,14 +421,39 @@ export class FdmSDK {
   }
 
   public async inspectInstances(inspectFilter: InspectFilter): Promise<InspectResultList> {
-    const data: any = inspectFilter;
-    const result = await this._sdk.post(this._inspectEndpoint, { data });
+    // Endpoint will soon have breaking changes, thus testing both new and old format
+    try {
+      const oldFormatInspectFilter: OldFormatInspectFilter = {
+        inspectionOperations: { involvedViewsAndContainers: {} },
+        items: inspectFilter.items ?? []
+      };
+      const result = await this._sdk.post(this._inspectEndpoint, { data: oldFormatInspectFilter });
+
+      if (result.status === 200) {
+        const oldFormatInspectResult = result.data as OldFormatInspectResultList;
+        return {
+          items: oldFormatInspectResult.items.map((item) => ({
+            ...item,
+            inspectionResults: {
+              involvedContainers: item.inspectionResults.involvedViewsAndContainers.containers,
+              involvedViews: item.inspectionResults.involvedViewsAndContainers.views
+            }
+          }))
+        };
+      }
+    } catch (e) {}
 
-    if (result.status === 200) {
-      return result.data as InspectResultList;
+    try {
+      const result = await this._sdk.post(this._inspectEndpoint, { data: inspectFilter });
+
+      if (result.status === 200) {
+        return result.data as InspectResultList;
+      }
+    } catch (e) {
+      throw new Error(`Failed to fetch instances`);
```

Comment:
I'd like to remove this `try`-block altogether, but we can wait until we remove the old endpoint call for good.